drivers/hmis/app/models/hmis/ce/match/candidate_pool_builder.rb

```diff
@@ -56,12 +60,13 @@ def update_pools!(values)
       end
       result = Hmis::Ce::Match::CandidatePool.import(
         attrs,
-        on_duplicate_key_update: {
```

Comment:
small bugfix: don't update `configuration_updated_at` if the row wasn't actually updated. Requires corresponding change to check for associated records when deleting orphan pools